app/registries/foreman/plugin.rb

```diff
@@ -608,6 +609,10 @@ def extend_allowed_registration_vars(var)
       @allowed_registration_vars << var
     end
 
+    def extend_observable_events(events)
+      (@observable_events << events).flatten!
```

Comment:
Should this also remove duplicates or is it the plugins' responsibility?

Reply:
Good idea, I can put `uniq` here for sure.